numpy/core/_methods.py

```diff
@@ -140,13 +141,18 @@ def _clip(a, min=None, max=None, out=None, *, casting=None, **kwargs):
         return _clip_dep_invoke_with_casting(
             um.clip, a, min, max, out=out, casting=casting, **kwargs)
 
-def _mean(a, axis=None, dtype=None, out=None, keepdims=False):
+def _mean(a, axis=None, dtype=None, out=None, keepdims=False, where=True):
```

Comment:
```suggestion
def _mean(a, axis=None, dtype=None, out=None, keepdims=False, *, where=True):
```

Reply:
Ah, this makes sure that `where` can only be set as keyword argument, not positional, right? Thanks for the hint!

Reply:
Exactly. We haven't done it in many places elsewhere because we only recently required python versions supporting this syntax.